OpenSourcePolitics/decidim-tou
# release: add wicked_pdf 2.6.3, allow admin to update a scope & bump to ruby 2.7.7

#### :tophat: Description
Downgrade `wicked pdf` to 2.6.3 & allow admin to update a scope with GeoJSON Data 

#### :pushpin: Related Issues
*Link your PR to an issue*
- Add fixes #176 and #178 on branch `master` 

#### Testing
**To test fix #176**
- Log in as admin
- Access Backoffice
- Answer survey
- Export a single answer
- Email in letter_opener should be present with PDF

**To test fix #178**
- As an admin
- Update a scope

Thank you!
